src/applications/personalization/view-dependents/containers/ViewDependentsApp.jsx

```diff
@@ -51,6 +52,9 @@ const mapStateToProps = state => ({
   manageDependentsToggle: toggleValues(state)[
     FEATURE_FLAG_NAMES.manageDependents
   ],
+  dependencyVerification: toggleValues(state)[
```

Comment:
I wonder if we should make the name for this prop a little more descriptive? Like `dependencyVerificationToggleValue` or something?

Reply:
Good idea 👍